### DISCUSSION TITLE
Controlling users audio

### DISCUSSION THREAD

**Discussion by MrJellyTot** [mozilla/hubs#3256] [General Questions & Discussion] [answered]
Hi All, new to hubs and have spent almost 5 hours googling and trying to ask on discord but I get 0 replies, hopefully get an answer here. 

So as I said, ive been googling on how to control users audio and found this thread on [github ](https://github.com/mozilla/hubs/issues/1853) -  I was wondering if any controls have been implemented yet / where I can find the controls, similar to the controls you get for video audio, specifically I wish to set user microphone audio to Pannernode, linear with a max distance of 10m.

Hopefully I get a response, I appreciate any and all help :)

**Comment by robinkwilson** [comment]
I'm afraid this is a little too specific for us to answer completely in the discussion thread since we're a small team, we will leave this discussion thread open for a while to see if anyone else has any feedback!

I'd try discord again, we don't guarantee an answer, but if our community members or one of the devs has time, they'll answer you! Good luck!

**Comment by elmau** [comment, marked as answer]
You can go to spoke and in the root scene properties you can override the audio settings.

![scene root](https://user-images.githubusercontent.com/14818268/98293145-97f17b80-1f8c-11eb-9aca-b14fec65e5e2.png)
